templates/single-quiz/pagination.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * The template for displaying the quiz pagination.
+ *
+ * @author   Automattic
+ * @package  Sensei
+ * @category Templates
+ * @version  3.15.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+global $sensei_question_loop;
+
+?>
+
+<div class="sensei-quiz-pagination">
+	<div class="sensei-quiz-pagination__links">
+		<?php
+		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- No need to escape the pagination links.
+		echo paginate_links(
+			/**
+			 * Filters the quiz questions paginate links args.
+			 *
+			 * @see https://developer.wordpress.org/reference/functions/paginate_links/
+			 *
+			 * @since 3.15.0
+			 *
+			 * @param array $args The pagination arguments.
+			 */
+			apply_filters(
+				'sensei_quiz_pagination_args',
+				[
+					'total'     => $sensei_question_loop['total_pages'],
+					'current'   => $sensei_question_loop['current_page'],
+					'format'    => '?quiz-page=%#%',
+					'type'      => 'list',
+					'mid_size'  => 1,
+					'prev_next' => false,
+				]
+			)
+		);
+		?>
+	</div>
+
+	<div class="sensei-quiz-pagination__actions">
+		<div class="sensei-quiz-pagination__button-link">
+			<button type="submit" name="quiz_reset" class="sensei-stop-double-submission has-background has-text-color">
+				<?php esc_attr_e( 'Reset', 'sensei-lms' ); ?>
+			</button>
+
+			<input type="hidden" name="woothemes_sensei_reset_quiz_nonce" id="woothemes_sensei_reset_quiz_nonce" value="<?php echo esc_attr( wp_create_nonce( 'woothemes_sensei_reset_quiz_nonce' ) ); ?>" />
+		</div>
+
+		<div class="sensei-quiz-pagination__button-link">
+			<button type="submit" name="quiz_save" class="sensei-stop-double-submission has-background has-text-color">
+				<?php esc_attr_e( 'Save', 'sensei-lms' ); ?>
+			</button>
+
+			<input type="hidden" name="woothemes_sensei_save_quiz_nonce" id="woothemes_sensei_save_quiz_nonce" value="<?php echo esc_attr( wp_create_nonce( 'woothemes_sensei_save_quiz_nonce' ) ); ?>" />
+		</div>
+
+		<div class="wp-block-buttons">
+			<?php if ( $sensei_question_loop['current_page'] > 1 ) : ?>
+				<div class="wp-block-button is-style-outline">
+					<a
+						href="<?php echo esc_attr( add_query_arg( 'quiz-page', $sensei_question_loop['current_page'] - 1 ) ); ?>"
+						class="wp-block-button__link button quiz-submit complete sensei-stop-double-submission"
+					>
+						<?php esc_attr_e( '&lt; Prev', 'sensei-lms' ); ?>
```

Comment:
Had to replace `Previous` with `Prev` to save some space. To understand the reasoning, check out the screenshots.

Reply:
Maybe I should stick with the `Previous` label and show the buttons on a new line?

Reply:
I think this may be a concern for translators. People probably won't know what "Prev" means in order to translate it. Another point related to translations is that "Previous" might translate to a longer word in another language, so it would be nice if we could get this to look good with longer strings. If both buttons don't always fit on one row then that seems fine.

Reply:
Makes sense!

> It would be nice if we could get this to look good with longer strings

Here's an easy to do suggestion:

<img width="678" alt="Screenshot on 2021-12-13 at 15-06-20" src="https://user-images.githubusercontent.com/1612178/145818785-f4422dd2-8608-44c4-ab0f-52b2e351a63a.png">

Let me know if this could work or we should explore other options.